### DISCUSSION TITLE
[Question] Map generation

### DISCUSSION THREAD

**Discussion by kari909** [organicmaps/organicmaps#7439] [Q&A] [answered]
Hi!

Why do you need to generate maps at all? I thought you just downloaded maps from open street map. Can you explain this process?

**Comment by Jean-BaptisteC** [comment, marked as answer]
You can find more information here -> https://github.com/organicmaps/organicmaps/blob/master/tools/python/maps_generator/README.md

**Comment by hb0nd** [comment]
> Hi!
> 
> Why do you need to generate maps at all? I thought you just downloaded maps from open street map. Can you explain this process?

simply: we strip out stuff that isn't used, include things like wikipedia descriptions, build a routing graph, and simplify geometry shapes.
